src/firebaseConfig.js

```diff
@@ -1,11 +0,0 @@
-const config = {
-  apiKey: "AIzaSyA45EPy0ZucETvoNMCp9dfxtb5yU44Kzio",
-  authDomain: "react-web-app-cource-a1413.firebaseapp.com",
-  databaseURL: "https://react-web-app-cource-a1413.firebaseio.com",
-  projectId: "react-web-app-cource-a1413",
-  storageBucket: "react-web-app-cource-a1413.appspot.com",
-  messagingSenderId: "642948747872",
-  appId: "1:642948747872:web:b71002891b7745c221a905"
-};
-
```

Comment:
What the cause of the deleting the firebase config?

Reply:
Config was added